src/tilecache.js

```diff
@@ -34,42 +34,54 @@
 
 (function( $ ){
 
-// private class
-var TileRecord = function( options ) {
-    $.console.assert( options, "[TileCache.cacheTile] options is required" );
-    $.console.assert( options.tile, "[TileCache.cacheTile] options.tile is required" );
-    $.console.assert( options.tiledImage, "[TileCache.cacheTile] options.tiledImage is required" );
-    this.tile = options.tile;
-    this.tiledImage = options.tiledImage;
-};
 
-// private class
-var ImageRecord = function(options) {
-    $.console.assert( options, "[ImageRecord] options is required" );
-    $.console.assert( options.data, "[ImageRecord] options.data is required" );
+// private class, but type required for docs
```

Comment:
You can mark it as `@private`. 

That said, it's returned by `getCache`, so it shouldn't be private. 

BTW, should we continue to call it `ImageRecord`? Seems like it's becoming more of a generic cache record?